pkg/seatbelts/seatbelt.go

```diff
@@ -0,0 +1,35 @@
+// Package seatbelt is where all the different types of seatbelt's structures will be written.
+package seatbelt
+
+import (
+	"fmt"
+)
+
+// Seatbelt represents a seatbelt type.
+type Seatbelt struct {
+	engaged bool
+}
+
+// NewSeatbelt function is used to create new instances of the Seatbelt structure
+func NewSeatbelt(specs map[string]bool) *Seatbelt {
```

Comment:
@juanct: this may be more user friendly with a simple:
func NewSeatbelt(engaged bool) *Seatbelt {